### DISCUSSION TITLE
[Develop a new way to insert images/videosfrom .jwlplaylist]: <Forced Suggestion due to new JW Library UPDATE>

### DISCUSSION THREAD

**Discussion by cong862023** [sircharlo/meeting-media-manager#2004] [Ideas] [answered]
### Is your idea related to a problem?

New update from JW Library (new: playlists)

### Describe your idea

A new update from the JW Library just came out. Now, playlists have been added, as we've been told in the past. 
It would be very useful to be able to identify and remove images and videos from playlists and insert them into the M3. With the update, a new extension, ".jwlplaylist", was created. 
So maybe it would be necessary to create or develop something to implement this function that would be excellent. 

Greetings from Portugal

### Describe alternatives you've considered

  .

**Comment by mtdvlpr** [comment, marked as answer]
Hi @cong862023, we'll definitely look into it! See #2003 for updates on this.

**Reply by cong862023** [reply]
👍

**Comment by 007brox** [comment]


**Reply by sircharlo** [reply]
Sorry, are you talking about M³?

**Reply by sircharlo** [reply]
I think you'll need to ask for support from "a friend who is familiar with JW Library" or from your "nearest branch office", as per the [official help documentation](https://www.jw.org/en/online-help/jw-library/android/faqs/). 😉
